src/components/tooltip/tests/Tooltip.spec.tsx

```diff
@@ -0,0 +1,76 @@
+import { shallow, mount, ReactWrapper } from 'enzyme';
+import { ITooltipProps, Tooltip } from '../Tooltip';
+// tslint:disable-next-line:no-unused-variable
+import * as React from 'react';
+
+describe('Tooltip', () => {
+  let tooltipWrapper: ReactWrapper<ITooltipProps, void>;
+  const TOOLTIP_PROPS: ITooltipProps = {
+    title: 'My test tooltip!'
+  };
+
+  describe('<Tooltip />', () => {
+    it('should render without error', () => {
+      expect(() => shallow(
+        <Tooltip {...TOOLTIP_PROPS}>
```

Comment:
`<Tooltip {...TOOLTIP_PROPS}>
    Hover me!
</Tooltip>` is repeated 3 times in the next 3 tests. Can we extract it to a variable or constant?